OracleDataIntegrator/README.md

```diff
@@ -36,30 +38,22 @@ Sample Data will look like this...
         ORACLE_PDB=odipdb
         ORACLE_PWD=Welcome1
         
-The Database 12.2.0.1 container can be started from the [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.2.0.1). 
-Follow the instructions to create a 12.2.0.1 based enterprise edition database or image can be pulled from OCR or DockerStore
-        
 Sample Command to Start the Database is as follows
 
          $ docker run --name ODI122126Database  -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/ODIVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
 
 The above command starts a DB container attaching to a network and mounting a host directory as /opt/oracle/oradata for persistence. 
 It maps the containers 1521 adn 5500 port to respective host port such that the services can be accessible outside of localhost.
 
-### ODI Distributable version 12.2.1.2.6 Docker image Creation and Running
+## ODI Distributable version 12.2.1.2.6 Docker image Creation and Running
 
 To build a ODI image either you can start from building Oracle JDK or use the already available Oracle JDK image.
 The Oracle JDK  image is not currently available.
 If plan to use available Oracle JDK image, skip next steps and jump to "Building Docker Image for ODI"
 
 ### Building Oracle JDK (Server JRE) base image
 
-Download the Oracle Server JRE binary and drop in folder `OracleJDK/java-8` and build that image.
-
-        $ cd OracleJDK/java-8
-        $ sh build.sh
-
-**NOTE:** The files to build JDK image can be found under [docker/OracleJDK](https://github.com/oracle/docker-images/tree/master/OracleJDK) or image can be pulled from OCR or DockerStore
+Oracle Server JRE image can be pulled from the [DockerStore](https://store.docker.com/images/oracle-serverjre-8) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleJava/java-8). For more information, visit the [OracleJava](../OracleJava) folder's [README](../OracleJava/README.md) file.
```

Comment:
As above, minor edit to change this to "Docker Store". I'd also add a space to [Oracle Java].